### PR TITLE
fixed daily links if there are no links

### LINKED ISSUE #948: Erreur accessing ?do=daily
I installed https://github.com/shaarli/Shaarli/releases/tag/v0.9.1 on a new server today and got the following error when I try to access ?do=daily:
```
Notice: Undefined offset: -1 in /var/www/shaarli/index.php on line 599

Fatal error: Uncaught Error: Call to a member function getTimestamp() on boolean in /var/www/shaarli/index.php:654 Stack trace: #0 /var/www/shaarli/index.php(869): showDaily(Object(PageBuilder), Object(LinkDB), Object(Shaarli\Config\ConfigManager), Object(PluginManager)) #1 /var/www/shaarli/index.php(2277): renderPage(Object(Shaarli\Config\ConfigManager), Object(PluginManager), Object(LinkDB), Object(History)) #2 {main} thrown in /var/www/shaarli/index.php on line 654
```

I have php v7.0.19-1 and apache 2.4.5 on debian 9
My time-zone is Europe/Paris

### PR DESCRIPTION
**Pull request by thewilli**
- the previous code tried to use links from a previous day if there are no one for the current one
- the new code skips this part if there are no entries (i.e. days) at all

fixes #948

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -588,7 +588,9 @@ function showDaily($pageBuilder, $LINKSDB, $conf, $pluginManager)
 
     $days = $LINKSDB->days();
     $i = array_search($day,$days);
-    if ($i===false) { $i=count($days)-1; $day=$days[$i]; }
+    if ($i===false && count($days)) {
+        $i=count($days)-1; $day=$days[$i];
```
Comment (virtualtam) [inline index.php]: we try to cleanup legacy code (most of it is in `index.php` ;-) ) as much as possible and stick to PSR recommendations:
- 1 instruction per line
- add spaces around operators

Comment (thewilli) [reply index.php]: @virtualtam ok, didn't knew that (just copy & pasted the original code and kept its format). I'll change it in the next days. Haven't heard of PSR yet (had to google it) - before the last PRs for Shaarli my last PHP experiences where at a time where PHP didn't yet support classes :wink:

Comment (virtualtam) [reply index.php]: No problem ;-)

The docs are a good starting point to start contributing to Shaarli:
- https://shaarli.readthedocs.io/en/master/Development-guidelines/
- https://shaarli.readthedocs.io/en/master/Static-analysis/
- https://shaarli.readthedocs.io/en/master/Unit-tests/

PSRs help (among other things) standardizing PHP code and avoid having as many coding styles as there are PHP projects...

Comment (thewilli) [reply index.php]: @virtualtam I updated the PR and modified the whole code of `showDaily()` to fit PSR-1 and PSR-2

**Pull request merged**